velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -66,7 +66,11 @@ int main(int argc, char** argv) {
           {"first", nullptr},
           {"first_ignore_null", nullptr},
           {"max_by", nullptr},
-          {"min_by", nullptr}};
+          {"min_by", nullptr},
+          // The skewness functions of Velox and DuckDB use different
```

Comment:
To disable verification against DuckDB, add the function to duckQueryRunner->disableAggregateFunctions call below.

Adding it here marks it as order-sensitive, which I assume it is not. Is it?